specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-04-01/examples/ManagedClustersCreate_Update.json

```diff
@@ -130,6 +139,9 @@
               ]
             }
           },
+          "windowsProfile": {
```

Comment:
Is there a repeat of the windowsProfile here ?

Reply:
It is not a repeat. One example is for 200 response, the other is for 201 response.